angular.json

```diff
@@ -0,0 +1,375 @@
+{
```

Comment:
@aitboudad 
Did you manually have to convert your .angular-cli.json to this, or did the upgrade do it auto.

Reply:
auto, `ng update @angular/cli` based on https://update.angular.io